test/e2e/auth/node_authn.go

```diff
@@ -60,6 +61,7 @@ var _ = SIGDescribe("[Feature:NodeAuthenticator]", func() {
 		pod := createNodeAuthTestPod(f)
 		for _, nodeIP := range nodeIPs {
 			// Anonymous authentication is disabled by default
+			nodeIP = getFormattedNodeIP(nodeIP)
 			result := framework.RunHostCmdOrDie(ns, pod.Name, fmt.Sprintf("curl -sIk -o /dev/null -w '%s' https://%s:%v/metrics", "%{http_code}", nodeIP, ports.KubeletPort))
```

Comment:
there is already a tool in the golang library
```suggestion
                        host := net.JoinHostPort(nodeIP, strconv.Itoa(ports.KubeletPort))
			result := framework.RunHostCmdOrDie(ns, pod.Name, fmt.Sprintf("curl -sIk -o /dev/null -w '%s' https://%s/metrics", "%{http_code}", host))
```

Reply:
Nice!, I'll use that way. Thanks @aojea